### ISSUE TITLE
ssh config includes don't work with wildcards / ~

### ISSUE THREAD

**Issue by akiessling** [Sequel-Ace/Sequel-Ace#1354]
- Sequel Ace Version: Version 3.4.5, Build 20025
- Sequel Ace Source: Homebrew
- macOS Version: 10.15.7
- Processor Type : Intel
- MySQL Version: not relevant
- macOS Localization: en

**Is Issue Present in [Latest Beta](https://github.com/Sequel-Ace/Sequel-Ace/releases)?**
yes

**Description**
Since my ssh config is quite large, i use the `Include` Feature and split the config up in a sub directory
```
Include ~/.ssh/config.d/*.config
```
Now each host that i connect to gets an own file and the main config file can be kept small and clean.
When Sequel Ace starts an ssh connection, it tries to use the Include, but does not find any files. I also added all of the files to the Preferences/Files, but this did not help.
The only way Include works is by specifying the complete path to a config: `Include /Users/andreas/.ssh/config.d/the-file.config` , using ~ or *.config in the Include fails

```
OpenSSH_8.1p1, LibreSSL 2.7.3
debug1: Reading configuration data /Users/andreas/.ssh/config
debug1: /Users/andreas/.ssh/config line 1: include ~/.ssh/config.d/*.config matched no files
```

**Steps To Reproduce**
1. Configure an Include in the main ssh config file with ~ and a wildcard for the filenames, add a ssh config to an included file
2. ssh to the server in the included file works, but Sequel Ace does not find the config

**Expected Behaviour**
The config should work like with ssh from the command line

**Comment by Jason-Morcos** [comment]
This may be a limitation of the sandbox, but I'm not quite sure (there may be some workaround). If someone wants to dig into this they're welcome to and I'd gladly review a PR, but this is likely going to be a limitation of Sequel Ace for the foreseeable future.

**Comment by luis-** [comment]
Yep looks like it's due to sandboxing. When Sequel Ace runs, it will have the `HOME` environment variable set (by macOS) to something like `/Users/<username>/Library/Containers/com.sequel-ace.sequel-ace/Data`. 

I believe that because of the above, when we run the ssh client, the `~` in `~/.ssh/config.d/*.config` will be expanded to: 
`/Users/<username>/Library/Containers/com.sequel-ace.sequel-ace/Data/.ssh/config.d/*.config`
 by ssh, instead of to the expected/usual:
`/Users/<username>/.ssh/config.d/*.config`. 

A work around would be to create a symbolic link form the `Data` directory back to the `.ssh` directory
`ln -s /Users/<username>/.ssh /Users/<username>/Library/Containers/com.sequel-ace.sequel-ace/Data/`

Tried this locally and it resolved the issue for me.

**Comment by Jason-Morcos** [comment]
Great investigation, @luis-! This workaround sounds like something we could/should include on our website documentation (held in /documentation on the repo)! A PR to do that from someone would be awesome!
